documentation/src/docs/getting-started.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Getting Started with Spark
+sidebar_label: Getting Started
+---
+
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+import TOCInline from '@theme/TOCInline';
+import { IfHaveFeature, IfMissingFeature } from 'nautilus-docs';
+
+<TOCInline toc={toc} />
+
+You can run Apache Spark applications written in Java, Scala, or Python. Using Python is the easiest to get started.
+
+## Prepare Development Environment
+
+<IfHaveFeature feature="nautilus">
+
+:::tip Streaming Data Platform
+SDP users can optionally skip this entire *Prepare Development Environment* section. However, it is often useful to develop applications locally in a sandbox environment before deploying to a production system such as SDP. For this reason, we recommend that this section is not skipped.
+:::
+
+</IfHaveFeature>
+
+### Install Operating System
```

Comment:
I think a `Prerequsites` section is better for OS and Java setups.

Reply:
Done. See prepare-development-environment.md. I did want to keep the specific steps so that users unfamiliar with Java do not struggle with installing JDK.